src/utils/bluetooth/useBluetooth.tsx

```diff
@@ -35,9 +35,7 @@ const useBluetooth = () => {
 
   const getState = async () => getBleManager().state()
 
-  const enable = async () => {
-    getBleManager().enable()
-  }
+  const enable = async () => getBleManager().enable()
```

Comment:
this is being awaited, but the function wasn't returning the promise 🤦

Reply:
Ah good catch